### PR TITLE
Update erb-linter template

### PR DESCRIPTION
**Pull request by coorasse**

### CODE REVIEW

File: templates/.erb-lint.yml [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
       Layout/InitialIndentation:
         Enabled: false
+      Layout/LineLength:
+        Enabled: true
       Layout/TrailingEmptyLines:
         Enabled: false
       Layout/TrailingWhitespace:
         Enabled: false
+      Layout/BlockAlignment:
+        Enabled: false
```
Comment (coorasse) [inline templates/.erb-lint.yml:23]: this has false-positives

Comment (CuddlyBunion341) [reply templates/.erb-lint.yml:23]: Could you please provide an example?
This seems to be a pretty important rule:
https://docs.rubocop.org/rubocop/cops_layout.html#layoutblockalignment

**Review by CuddlyBunion341 — APPROVED** [review]

File: templates/.erb-lint.yml [hunk 2]
```diff
@@ -1,20 +1,26 @@
 ---
-glob: "**/app/views/**/*.{html,text,js}{+*,}.erb"
 EnableDefaultLinters: true
+exclude:
+  - '**/rails_admin/*'
+  - "vendor/**/*"
 linters:
+  SpaceInHtmlTag:
+    enabled: false
```
Comment (simon-isler) [inline templates/.erb-lint.yml:8]:
```suggestion
    enabled: false
    NoJavascriptTagHelper:
    	enabled: true
	correction_style: 'plain'
```

https://github.com/Shopify/erb_lint/tree/main?tab=readme-ov-file#nojavascripttaghelper

**Review by simon-isler — COMMENTED** [review]
nice, could you also link the template in the markdown? I think I was missing this

File: templates/.erb-lint.yml (hunk 1 above)
Comment (coorasse) [reply templates/.erb-lint.yml:23]: I cannot write:
CleanShot 2024-09-04 at 17 22 27@2x

I'd have to write:
CleanShot 2024-09-04 at 17 22 49@2x

File: templates/.erb-lint.yml (hunk 2 above)
Comment (coorasse) [reply templates/.erb-lint.yml:8]: I am not sure that I understand this correctly.

Comment (simon-isler) [reply templates/.erb-lint.yml:8]: the indentation is a bit off. I'd add the `NoJavascriptTagHelper` with `correction_style: 'plain'`. the default adds  //<![CDATA[ and //]]> markers (see https://github.com/Shopify/erb_lint/tree/main?tab=readme-ov-file#nojavascripttaghelper)

**Comment by simon-isler** [comment]
> nice, could you also link the template in the markdown? I think I was missing this

👀

**Pull request merged**